Source/WebCore/dom/Element.cpp

```diff
@@ -1164,15 +1164,15 @@ void Element::scrollBy(const ScrollToOptions& options)
     ScrollToOptions scrollToOptions = normalizeNonFiniteCoordinatesOrFallBackTo(options, 0, 0);
     scrollToOptions.left.value() += scrollLeft();
     scrollToOptions.top.value() += scrollTop();
-    scrollTo(scrollToOptions, ScrollClamping::Clamped, ScrollSnapPointSelectionMethod::Directional);
+    scrollTo(scrollToOptions, ScrollClamping::Clamped, ScrollSnapPointSelectionMethod::Directional, ScrollClass::ScrollBy);
```

Comment:
As far as I can tell, `scrollToOptions` at this point contains a specific destination, so this is is a `ScrollTo` at this point. Is there any case in which `scrollToOptions.left` represents a delta, not an absolute value?

Reply:
This was intended to be passed down into AsyncScrollingCoordinator::requestScrollToPosition to indicate whether we should send across the delta or scroll position. This has been made more clear by passing down the original delta in the options as well.